components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/utils/mappings/APIMappingUtil.java

```diff
@@ -487,6 +487,11 @@ private static void setEndpointSecurityFromApiDTOToModel(APIDTO dto, API api) {
         APIEndpointSecurityDTO securityDTO = dto.getEndpointSecurity();
         if (dto.getEndpointSecurity() != null && securityDTO.getType() != null) {
             api.setEndpointSecured(true);
+            if(APIEndpointSecurityDTO.TypeEnum.KERBEROS.equals(securityDTO.getType())){
```

Comment:
format the if condition and remove the new line.

Reply:
fixed

Reply:
Swap the condition as `(securityDTO.getType(). equals (APIEndpointSecurityDTO.TypeEnum.KERBEROS))`